prime-router/src/test/kotlin/CsvFileTests.kt

```diff
@@ -108,7 +106,7 @@ class CsvFileTests {
         assertThat(settings.receivers).hasSize(2)
         assertThat(
             settings.findReceiver("federal-test.receiver")?.jurisdictionalFilter
-        ).isNotNull().hasSize(2)
+        ).isNotNull().hasSize(1)
```

Comment:
Why did this change?

Reply:
The tests on this file used the removed CSV defaults, so to keep it working I moved those defaults to be on federal test instead. The non-default juris filter for federal-test only has one criteria, which is why it's smaller.

You can see the relevant change on `test-organizations.yml`